### PR TITLE
Docs and Linting changes

### PR DESCRIPTION
**Pull request by RichardLitt**
All relatively minor. Let me know what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,11 +62,23 @@ See [example/index.html](https://github.com/haadcode/orbit-db-counterstore/blob/
...
+
+Please, feel free to contribute! Take a look at [the issues](https://github.com/orbitdb/orbit-db-identity-provider/issues), and comment on an existing issue or create a new one if you have questions, bugs, or suggestions. For larger PRs, open an issue first if you could - drive-by PRs are also welcomed.
 
-## Contributing
+Please abide by the [Code of Conduct](CODE_OF_CONDUCT.md). For more on contributing to OrbitDB, check out the docs in [orbitdb/welcome](https://github.com/orbitdb/welcome).
 
-See [orbit-db's contributing guideline](https://github.com/haadcode/orbit-db#contributing).
+See [OrbitDB's contributing guideline](https://github.com/haadcode/orbit-db#contributing).
+
+### Linting
```
Comment (haadcode) [inline README.md]: Do we need a separate section for linting, or is it enough to provide the link to the contribution guidelines (which (should) contain linting instructions)?

**Comment by haadcode** [comment]
Thanks, looks good 👍 One question re. adding linting section to readme.

File: README.md (hunk 1 above)
Comment (RichardLitt) [reply README.md]: Well, we _should_ also have local contribution guidelines that should have this information. Linking to the contribution guidelines should work. I'll add a section for linting to them. Look at the edits I suggest - what do you think?

Comment (haadcode) [reply README.md]: I meant if linting is worth it's own paragraph if all the information is in the guidelines? I think it's not.

Comment (RichardLitt) [reply README.md]: I disagree; it's in the package.json, but this paragraph is basically for people who haven't dealt with this repository before who may not know to run `lint` before submitting a PR. It's a tiny bit of duplication to save us having to say "run lint, please".

**Comment by RichardLitt** [comment]
@haadcode Updated.

**Pull request merged**